examples/text_summarization/extractive_summarization_cnndm_transformer.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Is this line supposed to be code, not a markdown?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
choose to make it markdown so that this part doesn't run for integration test.